pkgs/by-name/et/ethercat/package.nix

```diff
@@ -15,7 +22,9 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "etherlab.org";
     repo = "ethercat";
     rev = "refs/tags/${finalAttrs.version}";
-    hash = "sha256-fvuyADZzH1lHo1TkGd9s3n2n8y9vsGgs/ZOakIw48WI=";
+    hash = "sha256-w1aoznLjCHh+dN2fyfpKpzq68B4D/rert2XFpTxk/F4=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
```

Comment:
Can't we mock the git information? this is now likely sometimes unreproducible

Reply:
The Git information is used in the documentation here:

https://github.com/NixOS/nixpkgs/blob/14a1b28727fac1532a2b21269c0f5f0d317e4405/pkgs/by-name/et/ethercat/package.nix#L61

I am not sure how I could mock the Git commit date or hash..